care/users/tests/test_change_password.py

```
@@ -1,51 +1,49 @@
+from care.utils.tests.base import CareAPITestBase
 
 
-class TestChangePassword(APITestCase):
+class UserChangePasswordTestCase(CareAPITestBase):
     def setUp(self):
-        self.user = User.objects.create_user(
-            username="vipul",
-            password="StrongPass@123",
+        super().setUp()
+        self.user = self.create_user_with_password(
+            username="testuser", password="password123"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)